cmd/gateway/azure/gateway-azure.go

```diff
@@ -572,7 +572,7 @@ func (a *azureObjects) ListObjects(ctx context.Context, bucket, prefix, marker,
 		}
 
 		for _, blobPrefix := range resp.BlobPrefixes {
-			if prefix == minio.GatewayMinioSysTmp {
+			if blobPrefix == minio.GatewayMinioSysTmp {
 				// We don't do strings.HasPrefix(blob.Name, minio.GatewayMinioSysTmp) here so that
 				// we can use tools like mc to inspect the contents of minio.sys.tmp/
 				// It is OK to allow listing of minio.sys.tmp/ in non-recursive mode as it aids in debugging.
```

Comment:
as per this comment, looks like it was deliberately left in so as to inspect minio.sys contents. Is this no longer required?

Reply:
@poornas the comment says (unlike the recursive listing) we don't do strings.HasPrefix (here we instead do `==`)

this is to support this output:
```
krishna@escape:~$ mc ls myazure/minio-js-test-20edb793-1e6b-4325-9f28-41070871be01/
[2018-05-07 16:49:55 PDT]    10B 259896e7-259e-4d36-bcd3-3f6693349218
[2018-05-07 16:49:51 PDT]     1B miniojsPrefix.0
[2018-05-07 16:49:51 PDT]     1B miniojsPrefix.1
[2018-05-07 16:49:51 PDT]     1B miniojsPrefix.2
[2018-05-07 16:49:51 PDT]     1B miniojsPrefix.3
[2018-05-07 16:49:51 PDT]     1B miniojsPrefix.4
[2018-05-07 16:49:51 PDT]     1B miniojsPrefix.5
[2018-05-07 16:49:51 PDT]     1B miniojsPrefix.6
[2018-05-07 16:49:51 PDT]     1B miniojsPrefix.7
[2018-05-07 16:49:51 PDT]     1B miniojsPrefix.8
[2018-05-07 16:49:51 PDT]     1B miniojsPrefix.9
[2018-05-07 18:52:20 PDT]     0B 1/
krishna@escape:~$ 
```

notice that there is no minio.sys.tmp in this output

but using mc we can inspect inside minio.sys.tmp:
```
krishna@escape:~$ mc ls myazure/minio-js-test-20edb793-1e6b-4325-9f28-41070871be01/minio.sys.tmp/multipart/v1/
[2018-05-07 18:52:58 PDT]     0B 161d957ee6363c88.1343f38db3f7a65f950f2c4967832cd8513bcb8aaaf7ba74782396e32405001b/
[2018-05-07 18:52:58 PDT]     0B 58d605dcea43eee2.8a1abc59b53c058c8e350ab98cec3ca6e7a4fa3375b1a161ffee0760a1cbac49/
[2018-05-07 18:52:58 PDT]     0B 8d74c1e33fae3ea4.4f094ed9ffbe187425ebd84ebd129683d55550943dd31f2bee38414c7e59661c/
krishna@escape:~$ 
```

Reply:
ok, missed that.